src/chef_authn.erl

```diff
@@ -83,7 +83,13 @@
                           {time, erlang_time() | now} |
                           {path, http_path()} |
                           {get_header, header_fun()}.
-
+-ifdef(otp_18_plus).
```

Comment:
👍  fine by me.  Let's merge this as it passed pedant.  If someone want to change this they can PR it.